processor/samplingprocessor/go.mod

```diff
@@ -50,3 +50,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/observiq/bindplane-agent/expr => ../../expr
```

Comment:
We'll also want to change the version of `github.com/observiq/bindplane-agent/expr` to v1.61.0 up top; 

You could do this by just running `make update-modules NEW_VERSION=v1.61.0`.